spec/features/applications/admin/show_spec.rb

```diff
@@ -0,0 +1,206 @@
+    pet = applicant.pets.create(name: "Scooby", age: 2, breed: "Great Dane", adoptable: true, shelter_id: shelter.id)
+
+    visit "/admin/applications/#{applicant.id}"
+
+    click_link("Scooby")
+
+    expect(current_path).to eq("/pets/#{pet.id}")
+  end
+
+  it "shows a pets search field" do 
+    shelter = Shelter.create(name: "Mystery Building", city: "Irvine CA", foster_program: false, rank: 9)
+    applicant = Application.create(name: "Shaggy", street_address: "123 Mystery Lane", city: "Irvine", state: "CA", zip_code: "91010", description: "Because Scoob and I love Scooby Snacks")
+    pet = applicant.pets.create(name: "Scooby", age: 2, breed: "Great Dane", adoptable: true, shelter_id: shelter.id)
+
+    visit "/admin/applications/#{applicant.id}"
+    
+    expect(page).to have_content("Search for a pet to add")
+    
+  end
+
+  
+  it "shows pets matching user search input" do 
+    shelter = Shelter.create(name: "Mystery Building", city: "Irvine CA", foster_program: false, rank: 9)
+    applicant = Application.create(name: "Shaggy", street_address: "123 Mystery Lane", city: "Irvine", state: "CA", zip_code: "91010", description: "Because ")
+    pet = Pet.create(name: "Scooby", age: 2, breed: "Great Dane", adoptable: true, shelter_id: shelter.id)
+
+    visit "/admin/applications/#{applicant.id}"
+    
+    fill_in "Search", with: "scoob"
+
+    click_button "Search"
+    
+    expect(page).to have_content("Scooby")
+    
+  end
+
+  it 'shows a link to adopt a pet under each shown pet' do 
+    shelter = Shelter.create(name: "Mystery Building", city: "Irvine CA", foster_program: false, rank: 9)
+    applicant = Application.create(name: "Shaggy", street_address: "123 Mystery Lane", city: "Irvine", state: "CA", zip_code: "91010", description: "Because ")
+    pet = Pet.create(name: "Scooby", age: 2, breed: "Great Dane", adoptable: true, shelter_id: shelter.id)
+
+    visit "/admin/applications/#{applicant.id}"
+    fill_in "Search", with: "scoob"
+    
+    click_button "Search"
+    
+    
+    expect(page).to have_button("Adopt Scooby")
+    
+  end
+
+  it "shows added pets to the application" do 
+    shelter = Shelter.create(name: "Mystery Building", city: "Irvine CA", foster_program: false, rank: 9)
+    applicant = Application.create(name: "Shaggy", street_address: "123 Mystery Lane", city: "Irvine", state: "CA", zip_code: "91010", description: "Because ")
+    pet = Pet.create(name: "Scooby", age: 2, breed: "Great Dane", adoptable: true, shelter_id: shelter.id)
+
+    visit "/admin/applications/#{applicant.id}"
+    fill_in "Search", with: "scoob"
+    
+    click_button "Search"
+    click_button "Adopt #{pet.name}"
+    
+    visit "/admin/applications/#{applicant.id}"
+    
+    fill_in "Search", with: "z"
+    click_button "Search"
+    expect(page).to have_content("Scooby")
+    
+  end
+
+  it "does NOT show a section to explain why I'd make a good owner if I have NOT chosen pets to adopt (US-7)" do
```

Comment:
Again, this functinality wasn't necessary to add to the admin side, but this is great testing! :)